src/gui/shell.cpp

```diff
@@ -1332,28 +1332,20 @@ void Shell::wheelEvent(QWheelEvent *ev)
 		return;
 	}
 
-	QPointF scroll_delta_f{ ev->angleDelta() / QWheelEvent::DefaultDeltasPerStep };
+	// Convert angleDelta() to QPointF then divide it with DefaultDeltasPerStep
+	QPointF scroll_delta_f{ ev->angleDelta() };
+	scroll_delta_f /= QWheelEvent::DefaultDeltasPerStep;
```

Comment:
No need to change this line, right?

This should be functionally equivalent to what existed before.

Reply:
Previous code divided angleDelta with that number BEFORE converting it from QPoint to QPointF, so there was an undesired rounding.

Reply:
Ahhh I see, thanks. Very subtle, `{}` usually catches such issues.

Let's call this out in a comment or write this in a way that clearly indicates the desired types.
`QPointF scroll_delta_f{ QPointF{ ev->angleDelta() } / QWheelEvent::DefaultDeltasPerStep };`?

Alternatively, move `scroll_delta_f` calculation small function + write some tests to prevent regression in the future. Perhaps overkill here.